adapter/persistence/latest_entry_repository.go

```diff
@@ -0,0 +1,68 @@
+package persistence
+
+import (
+	"github.com/pkg/errors"
+	"github.com/utahta/momoclo-channel/domain"
+	"github.com/utahta/momoclo-channel/domain/entity"
+)
+
+type LatestEntryRepository struct {
+	DatastoreHandler
+}
+
+func NewLatestEntryRepository(h DatastoreHandler) *LatestEntryRepository {
+	return &LatestEntryRepository{h}
+}
+
+func (repo *LatestEntryRepository) Save(l *entity.LatestEntry) error {
+	return repo.RunInTransaction(func(h DatastoreHandler) error {
+		return h.Put(l)
+	}, nil)
+}
+
+func (repo *LatestEntryRepository) FindByURL(urlStr string) (*entity.LatestEntry, error) {
+	const errTag = "LatestEntryRepository.FindByURL failed"
+
+	l := &entity.LatestEntry{ID: entity.ParseLatestEntryCode(urlStr)}
+
+	err := repo.Get(l)
+	if err == domain.ErrNoSuchEntity {
+		return nil, err
+	}
+	if err != nil {
+		return nil, errors.Wrap(err, errTag)
+	}
+	return l, nil
+}
+
+func (repo *LatestEntryRepository) getURL(code string) string {
+	l := &entity.LatestEntry{ID: code}
+	if err := repo.Get(l); err != nil {
+		return ""
+	}
+	return l.URL
+}
+
+func (repo *LatestEntryRepository) GetTamaiURL() string {
+	return repo.getURL(entity.LatestEntryCodeTamai)
+}
+
+func (repo *LatestEntryRepository) GetMomotaURL() string {
+	return repo.getURL(entity.LatestEntryCodeMomota)
+}
+
+func (repo *LatestEntryRepository) GetAriyasuURL() string {
+	return repo.getURL(entity.LatestEntryCodeAriyasu)
+}
+
+func (repo *LatestEntryRepository) GetSasakiURL() string {
+	return repo.getURL(entity.LatestEntryCodeSasaki)
+}
+
+func (repo *LatestEntryRepository) GetTakagiURL() string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method LatestEntryRepository.GetTakagiURL should have comment or be unexported